rpcs3/Emu/RSX/Overlays/overlay_perf_metrics.cpp

```diff
@@ -334,9 +386,12 @@ namespace rsx
 					total_cycles = ppu_cycles + spu_cycles + rsx_cycles;
```

Comment:
Would `total_cycles = std::max<u64>(ppu_cycles + spu_cycles + rsx_cycles, 1)`; work here as well? IMO it is better because it doesn't depend on ppu/spu/rsx_usage variables initial value and simplifies logic. only a suggestion.

Reply:
fixed